### DISCUSSION TITLE
More branding options

### DISCUSSION THREAD

**Discussion by sboklund** [retypeapp/retype#188] [Ideas]
Hi,

First of all, thank you very, very much for this wonderful piece of software!

Our company is in the process of implementing a Retype-project and we´re soo pleased with everything about it, but would love to be able to sprinkle just a tiny bit more branding on top. What we would like to see is:

- More and/or custom font options (just echoing this request)
- Custom color configuration extended to (all) links

Friendly suggestion: add these as pro-features – we´d be happy to pay!

Best regards,
Sebastian

**Comment by geoffreymcgill** [comment]
Hi @sboklund. Thanks for the kind words and for giving Retype a try. 

We are deep into add new branding options to Retype and anticipate releasing those features with `v1.12`. 

I do have a question regarding your request for font options. 

> More and/or custom font options (just echoing this request)

Can you describe the scenario for why you require changing the font? Basically, why do you need to change the font and what font would you like to change too? Would you require changing any other  characteristics of the font, such as size, weight, spacing, etc?

**Reply by sboklund** [reply]
Hi Geoffrey,

Thanks for your reply, happy to hear that more branding options are coming soon!

>Can you describe the scenario for why you require changing the font? Basically, why do you need to change the font and what font would you like to change too?

To better align with our graphic design profile and give a "cohesive" impression when moving between our various sites/systems.

Ideally, we would like to change to: `font-family: "Open Sans", Arial, Helvetica, sans-serif;`, as this is what we use elsewhere.

>Would you require changing any other characteristics of the font, such as size, weight, spacing, etc?

Not necessarily. After playing around in the dev tools a bit, it appears that the size, weight, spacing etc. looks OK for the configuration mentioned above. 

On the other hand, I fully understand if you might not want to open up for users to freely customize this. Perhaps it could make sense to add support for a select few fonts (2-5?), test them out and adjust size, weight etc. to an extent that you feel happy with, and then offer this subset as a config option in Retype Pro? Of course, I´m just thinking out loud here :)

Cheers,
Sebastian
